tests/src/Kernel/Entity/OgMembershipTest.php

```diff
@@ -61,10 +64,14 @@ protected function setUp() {
     $this->installConfig(['og']);
     $this->installEntitySchema('og_membership');
     $this->installEntitySchema('entity_test');
+    $this->installEntitySchema('node');
     $this->installEntitySchema('user');
     $this->installSchema('system', 'sequences');
 
     $this->entityTypeManager = $this->container->get('entity_type.manager');
+    $storage = $this->entityTypeManager->getStorage('user');
+    // Insert a row for the anonymous user.
+    $storage->create(['uid' => 0, 'status' => 0, 'name' => ''])->save();
```

Comment:
Can you add a line of documentation here that refers to `@see user_install()`, so that it is clear why this is necessary?